### ISSUE TITLE
Design and Implement the Contact tab

### ISSUE THREAD

**Issue by lalitmee** [GirlScriptSummerOfCode/Kranti#19]
For requirements, refer to https://drive.google.com/drive/folders/0BwpCIeAR0xFcTVlXc1FvQXZvcHM?usp=sharing

Try to come up with a simple and clean design of the Contact tab. :+1: 
For any help regarding any query contact in the channel `Kranti`.

**Comment by devesh-verma** [comment]
I would like to work on this. Plz assign it to me.

**Comment by AvaniVerma** [comment]
No. You haven't completed the issue already assigned to you.

**Comment by lalitmee** [comment]
@devshiva619, did you complete the last assigned issue?

**Comment by devesh-verma** [comment]
@lalitmee only minor change. I will produce the PR my tonight for that and for this by tomorrow or day after.

**Comment by ayush221b** [comment]
Can you please give me this issue?

**Comment by lalitmee** [comment]
@devshiva619, until and unless your last PR related to the assigned issue is not merged, you can't be assigned to another issue.

**Comment by AvaniVerma** [comment]
@ayush221b I've already assigned an issue to you. So you can't work on this one.

**Comment by shreyateeza** [comment]
@AvaniVerma  I would like to work on this issue.

**Comment by adi44** [comment]
I will do it

**Comment by saberprashant** [comment]
I would like to work on this one, so can anyone assign this to me? @AvaniVerma  @lalitmee @troublemagnet

**Comment by lalitmee** [comment]
@shreyateeza, @adi44, and @saberprashant do you guys have any other issue pending?

**Comment by shreyateeza** [comment]
@lalitmee Yes I am working on issue #17. But would like to work on this as well if possible.

**Comment by saberprashant** [comment]
@lalitmee  No, I have no issue pending, the last issue on which I was working was "Designing a logo" and I submitted that already.

**Comment by lalitmee** [comment]
@shreyateeza, if this is the case, I would suggest you finish that first. If you want help with your previous issue, contact us on the slack channel. :+1: 
@saberprashant, I think you can work on this. I hope you know **React.js**. Because till now I haven't seen anyone using the correct way of React.js Development in our project. :worried:

**Comment by saberprashant** [comment]
@lalitmee , Thanks for assigning me this one,  I will try my best to implement it with react standards and first I will provide a basic design of what I am going to implement in this tab, so you will also get the idea how I am going to do it. So tomorrow around 12pm, I will mention the design here. :)

**Comment by adi44** [comment]
I dont have any issues pending pls assign some issue

**Comment by saberprashant** [comment]
@lalitmee  @troublemagnet  @AvaniVerma,  
Hi there,  
Well, I made a basic design for the Contact tab as per your requirements(drive link). I tried to keep it simple and to provide everything in a single page view. Also, this design is just a pointer of what I am thinking to make, so your suggestions are most welcome :)  
[--View the design here--](https://drive.google.com/file/d/1gZDvO1Hm8WB-oyT1I6dN1kX_Yt1J9vRy/view?usp=sharing)

**Comment by AvaniVerma** [comment]
@saberprashant Design looks good to me but the final decision will depend on the implementation. So go ahead and code it.

**Comment by saberprashant** [comment]
@AvaniVerma , Okay, I will start implementing this and will submit a PR.

**Comment by lalitmee** [comment]
@saberprashant, yeah design is good. try to make it as much as clean and good looking. :+1:

**Comment by saberprashant** [comment]
Hi there @lalitmee, I see that no one worked on the back-end for this project yet, so, can I make the database for the contact tab on m-lab? So, that I can connect the part with the node.js for the updates form.

**Comment by lalitmee** [comment]
@saberprashant, I don't think that there is need of database.. only the thing we need to do is to set up some email contact and mobile notification thing so that some person from the organization can know that someone wants to connect with them

**Comment by saberprashant** [comment]
Then I think, we can use firebase for that.

**Comment by lalitmee** [comment]
okk, you can do whatever you want to do.. and create a PR

**Comment by saberprashant** [comment]
Okay then, I will have to change the form accordingly. Will submit a PR by
the end of this Sunday :)

**Comment by saberprashant** [comment]
Hi @lalitmee , can you please merge the old pull requests of other contributors related to tabs and other, so that I can connect my code with that, as currently, I have nothing to pull from the code base. Plus, I need to add the firebase, so it would be better if you will create a firebase account in name of Kranti(email).  
Thanks!

**Comment by troublemagnet** [comment]
@saberprashant I closed the PR related to tabs due to inactivity, please feel free to make one yourself.
